### PR TITLE
Based on branch LumiSetup: Scripts updated to run a large model on LUMI (and some infer scripts which are not working)

### PR DESCRIPTION
**Pull request by inakbk**
**Changes include:**
- Files to make environment with container, see  `/make_env`
- Script to see job stats like GPU usage on LUMI while job is running: `monitor_interactive.sh` (taken from `aifs-mono`)
- Training scripts:
    - Working training script `lumi_train.sh`. Most important was to add correct directory bindings (-B). Still need to make `CPU_BIND` work
    - See summary of changes in `main.yaml` below
    - For convenienve there is also a `main_temp.yaml`, but this may be deleted if you want @mateuszmatu ?
- Infer scripts which are not working yet include `lumi_infer.sh`, `run_pytorch_infer.sh` and a config file in `/inference`

**Changes in `main.yaml` include:**

> num_workers: 2

and also setting `limit_batches` to `null` (previous value of 10 is good for debugging). 
In order to make the training work we had to start using "model paralell", which is the same as setting 

> num_gpus_per_model: 4

and also

> num_gpus_per_node: 8

other changes include: graph res7, num_channels, learning rate, max_steps

### CODE REVIEW

File: lumi_train.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+CONFIG_DIR=$(pwd -P)/training/
+CONTAINER=$PROJECT_DIR/container/ocean-ai.sif
+VENV=/pfs/lustrep2/projappl/project_465001629/python-envs/anemoi-env-trimedge
+export VIRTUAL_ENV=$VENV
+
+module load LUMI/24.03 partition/G
+export SINGULARITYENV_LD_LIBRARY_PATH=/opt/ompi/lib:${EBROOTAWSMINOFIMINRCCL}/lib:/opt/cray/xpmem/2.4.4-2.3_9.1__gff0e1d9.shasta/lib64:${SINGULARITYENV_LD_LIBRARY_PATH}
+
+# MPI + OpenMP bindings: https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/distribution-binding
+CPU_BIND="mask_cpu:fe000000000000,fe00000000000000,fe0000,fe000000,fe,fe00,fe00000000,fe0000000000"
```
Comment (mateuszmatu) [inline lumi_train.sh:27]: denne linjen kan man strengt tatt fjerne

**Review by mateuszmatu — APPROVED** [review]
